File: abc/abc307/abc307_a/abc307_a.py
n = int(input())

a = list(map(int, input().split()))

cnt = 0

sum = 0

for i in range(len(a)):

  cnt += 1

  sum += a[i]

  

  if cnt == 7:

    print(sum, end = ' ')

    sum = 0

    cnt = 0

File: abc/abc307/abc307_a/abc307_a.java
import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int n = stdIn.nextInt();
    int[] a = new int[7 * n];
    for(int i = 0; i < 7 * n; i++) a[i] = stdIn.nextInt();

    int cnt = 0;
    int sum = 0;

    for(int i = 0; i < a.length; i++) {
      cnt++;
      sum += a[i];

      if(cnt == 7) {
        System.out.printf("%d ", sum);
        cnt = 0;
        sum = 0;
      }
    }
  }
}